src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Climber extends SubsystemBase {
+
+    private CANSparkMax liftMotor;
+    private CANSparkMax yoyoMotor;
+
+    public Climber() {
+        liftMotor = new CANSparkMax(Constants.CLIMBER_LIFT_MOTOR_PORT, MotorType.kBrushless);
+        yoyoMotor = new CANSparkMax(Constants.CLIMBER_YOYO_MOTOR_PORT, MotorType.kBrushless);
+    }
+
+    public void climbUp() {
+        liftMotor.set(1.0);
```

Comment:
The speeds for climbUp and climbDown should be a constant that we can potentially change if needed